core/src/main/java/nl/nn/adapterframework/unmanaged/SpringJmsConnector.java

```diff
@@ -368,4 +368,3 @@ public String getLogPrefix() {
 
 
 }
-
```

Comment:
En deze newline eigenlijk ook...

Reply:
Wat mij betreft zouden we ons hier niet druk over moeten maken, gezien de staat van veel andere code en de dingen die daarin mis zijn. Just 2 two cents 😉

Reply:
Er staat bij mij nog steeds een newline aan het einde van de file... 😳

Reply:
Het gaat mij puur om de git history die onnodig 'vervuild' wordt + het houd het pr wat schoner :)